src/kernel/blockmanager_opts.h

```diff
@@ -27,6 +28,14 @@ struct BlockManagerOpts {
     bool fast_prune{false};
     const fs::path blocks_dir;
     Notifications& notifications;
+    const fs::path block_tree_db_dir;
+    // Whether to wipe the block tree database when loading it. If set, this
+    // will also set a reindexing flag so any existing block data files will be
+    // scanned and added to the database.
+    bool wipe_block_tree_db{false};
```

Comment:
Assuming this is just a comment explaining what was already happening and documenting a behaviour change introduced in this PR? (note to self to test this).

Reply:
Nevermind, I can see this a move only from `src/chainstate.h`